src/protobuf-net.Reflection/CodeGenerator.cs

```diff
@@ -599,6 +599,7 @@ internal GeneratorContext(CommonCodeGenerator generator, FileDescriptorProto fil
 
                 EmitNullWrappers = IsEnabled("nullwrappers", true);
                 EmitCompatibilityLevelAttribute = IsEnabled("compatlevel", true);
+                RepeatedAsList = IsEnabled("repeatedaslist");
 
```

Comment:
can you take a look at the additional bits needed here in https://github.com/protobuf-net/protobuf-net/commit/52fc07afe51c9f94f8d529b5a880d487e601df19 please? in particular, look at `protobuf-net.BuildTools.props` and `ProtoFileGenerator.cs` in that PR, which is how this option becomes available in the tooling; and ideally also the `Program.cs` bit to advertise the command-line feature